### PR TITLE
Travis

### PR DESCRIPTION
**Pull request by rspieldenner**

### CODE REVIEW

File: sstable-adaptor-cassandra/build.gradle [hunk 1]
```diff
@@ -35,7 +34,6 @@ dependencies {
     compile "com.googlecode.json-simple:json-simple:1.1"
     compile "com.boundary:high-scale-lib:1.0.6"
     compile "com.github.jbellis:jamm:0.3.0"
-    //compile "org.yaml:snakeyaml:1.11"
     compile "org.jacoco:org.jacoco.agent:0.7.5.201505241946"
```
Comment (rspieldenner) [inline sstable-adaptor-cassandra/build.gradle]: why are their compile dependencies on jacoco?

Comment (timiblossom) [inline sstable-adaptor-cassandra/build.gradle]: Fixed - thanks @rspieldenner

Comment (timiblossom) [inline sstable-adaptor-cassandra/build.gradle]: I will merge this manually

**Comment by timiblossom** [comment]
Manually merge

**Pull request closed without merging**